restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/SecuredEndpoint.java

```diff
@@ -331,6 +331,9 @@ private AuthOutcome secureLevelControl(Account operatedAccount, String resourceA
         String operatedAccountSid = null;
         if (operatedAccount != null)
             operatedAccountSid = operatedAccount.getSid().toString();
+        if (isSuperAdmin()) {
```

Comment:
will this affect to other existing operations? Seems like a big change.... Of course this means is the superAdmin anyway, but just wanted to check if this will change anything for other existing RESTAPI operations...

Reply:
this change would mean that superadmin account would have control over restcomm APIs (number, clients, call, conference etc) regardless of the fact that resource being owned by another account in the system

Reply:
@maria-farooq but there is already a method for this `org.restcomm.connect.http.SecuredEndpoint#isSuperAdmin` and can be used from any endpoint its needed.

I think if we go with this patch then we mess the SecuredEndpoints and the rules to follow and it will be harder to debug later.

I think we should remove this and instead check with `isSuperAdmin` method

Reply:
@gvagenas good point, i added this check for superadmin in conference and call endpoints intead